### PR TITLE
[fix] Payments table on the invoice is showing in the print, even if IS POS option is disabled

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
Issue
screen shot 2017-05-12 at 6 23 43 pm

Fixed
screen shot 2017-05-12 at 6 24 32 pm

### CODE REVIEW

File: erpnext/patches/v8_0/make_payments_table_blank_for_non_pos_invoice.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import frappe
+
+def execute():
+	frappe.reload_doctype('Sales Invoice')
+
+	frappe.db.sql("""
+		delete from 
+			`tabSales Invoice Payment` 
+		where 
+			parent in (select name from `tabSales Invoice` where is_pos = 0)
```
Comment (nabinhait) [inline erpnext/patches/v8_0/make_payments_table_blank_for_non_pos_invoice.py]: To be in safe side, add another condition for payment amount

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/patches/v8_0/make_payments_table_blank_for_non_pos_invoice.py (hunk 1 above)
Comment (rohitwaghchaure) [inline erpnext/patches/v8_0/make_payments_table_blank_for_non_pos_invoice.py]: Fixed

Comment (rohitwaghchaure) [inline erpnext/patches/v8_0/make_payments_table_blank_for_non_pos_invoice.py]: Revert the changes
